src/Handler/RedisHandler.php

```diff
@@ -69,25 +69,25 @@ public function __construct(\Redis $redis, array $options = [])
 	 */
 	public function close()
 	{
-		$this->redis->close();
-
+		// No need to close the connection to Redis server manually.
 		return true;
 	}
 
 	/**
-	 * Destroy a session
+	 * Destroy a session, called automatically when running session_regenerate_id().
 	 *
 	 * @param   integer  $session_id  The session ID being destroyed
 	 *
 	 * @return  boolean  True on success, false otherwise
 	 *
 	 * @since   2.0.0
 	 */
-	public function destroy($session_id)
+	public function destroy($session_id): bool
```

Comment:
If we're doing this should make this type hint for all handlers rather than just this handler

Reply:
Done.